src/scripts/game.js

```diff
@@ -142,7 +153,11 @@ class Game {
   }
 
   addFlowers() {
-    return new Flower({ pos: this.randomPosition(), game: this });
+    let positions = this.flowerPosGenerator();
```

Comment:
Renaming the function to more of a verb will read a little better - e.g. `let positions = this.generateFlowerPositions();`